features/features.go

```diff
@@ -548,4 +548,10 @@ var (
 							productScope(ocpSpecific).
 							enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 							mustRegister()
+	FeatureGateImageStreamImportMode = newFeatureGate("ImageStreamImportMode").
+                                                reportProblemsToJiraComponent("Multi-Arch").
+                                                contactPerson("psundara").
+                                                productScope(ocpSpecific).
+                                                enableIn(configv1.TechPreviewNoUpgrade).
```

Comment:
You always need DevPreviewNoUpgrade if you have TechPreviewNoUpgrade

Reply:
done.